Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -451,11 +456,18 @@ public struct ConvertAction: Action, RecreatingContext {
             throw error
         }
 
-        var allProblems = analysisProblems + conversionProblems
-
-        if allProblems.containsErrors == false {
-            let coverageResults = try coverageAction.perform(logHandle: logHandle)
-            allProblems.append(contentsOf: coverageResults.problems)
+        var didEncounterError = analysisProblems.containsErrors || conversionProblems.containsErrors
+        if try context.renderRootModules.isEmpty {
+            postConversionProblems.append(
+                Problem(
+                    diagnostic: Diagnostic(
+                        severity: .warning,
+                        identifier: "org.swift.docc.MissingTechnologyRoot",
+                         summary: "No TechnologyRoot to organize article-only documentation.",
+                         explanation: "Article-only documentation need a TechnologyRoot page (indicated by a `TechnologyRoot` directive within a `Metadata` directive) to define the root of the documentation hierarchy."
```

Comment:
```suggestion
                         explanation: "Article-only documentation needs a TechnologyRoot page (indicated by a `TechnologyRoot` directive within a `Metadata` directive) to define the root of the documentation hierarchy."
```

I think since 'documentation' is an uncountable noun, 'needs' should be pluralized like this.

Reply:
Thanks for pointing this out @krilnon, fixed in https://github.com/apple/swift-docc/pull/664/commits/9aaa7f889fef9c9319f7effbd400b9d573caea46